<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadFolderWidget</class>
 <widget class="QWidget" name="LoadFolderWidget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="loadFolderImage">
     <property name="title">
      <string>Load Folder Image</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0" colspan="2">
       <widget class="QGroupBox" name="imageOrgine">
        <property name="title">
         <string>Image Ultrasound</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="0" colspan="3">
          <widget class="QListView" name="listView"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="listImageOrigine">
           <property name="text">
            <string>List: </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="buttonRunImage">
           <property name="text">
            <string>Load Ultrasound</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="boxListImageOrigine">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QGroupBox" name="loadUnet">
        <property name="title">
         <string>Image Segmented</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="1">
          <widget class="QPushButton" name="buttonRunUnet">
           <property name="text">
            <string>Load Segmented</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>List:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="boxImageUnet"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
